storage/store.go

```diff
@@ -1324,8 +1319,7 @@ func (s *Store) ProposeRaftCommand(idKey cmdIDKey, cmd proto.InternalRaftCommand
 		panic("proposed a nil command")
 	}
 	// Lazily create group. TODO(bdarnell): make this non-lazy
-	err := s.multiraft.CreateGroup(uint64(cmd.RaftID))
-	if err != nil {
+	if err := s.multiraft.CreateGroup(uint64(cmd.RaftID)); err != nil {
```

Comment:
My understand of the rule was to not change those things in the same commit as other, orthogonal changes. In this case, I think this is cleaner because it doesn't declare `err` for the code that follows, and this line is short.